.github/workflows/e2e-test.yml

```diff
@@ -51,3 +51,15 @@ jobs:
       env:
         APP_CHECK_DEBUG_TOKEN: ${{ secrets.APP_CHECK_DEBUG_TOKEN }}
       run: xvfb-run yarn test:compat
+    - name: Tests succeeded
+      if: success()
+      run: |
+        curl -d "{\"text\": \"E2E Tests Succeded - https://github.com/${{github.repository}}/actions/runs/${{github.run_id}} \"}" -H "Content-Type: application/json" $WEBHOOK_URL
```

Comment:
Can we include the commit info and a link to the corresponding PR and release info if available directly in the message?

Reply:
Need to separate this out into a node script to parse more data, testing now.

Reply:
Ok, it's in a Node script now which allows more future flexibility I think.